readthedocsext/theme/templates/projects/partials/header.html

```diff
@@ -177,13 +177,61 @@
       </a>
       {% if request.user|is_admin:project %}
         <div class="right menu">
+          {% if user.is_staff %}
+            <a class="item" data-bind="click: $root.show_modal('project-debug')">
+              <i class="fa-duotone fa-microscope icon"></i>
+              {% trans "Debug" %}
+            </a>
+          {% endif %}
           <a class="item {{ edit_active }}" href="{% url "projects_edit" project.slug %}">
             <i class="fa-duotone fa-gears icon"></i>
             {% trans "Settings" %}
           </a>
         </div>
       {% endif %}
     </div>
+
+
+    <div class="ui mini modal" data-modal-id="project-debug">
+      <div class="header">
+        {% trans "Debug information" %}
+      </div>
+      {% comment %}
+        Show spam information on community instance.
+        Allows us to quickly understand what's the spam score,
+        and also know what rules are matching this project.
+      {% endcomment %}
+      <div class="content">
+        {% if not ALLOW_PRIVATE_REPOS %}
+          <h3>Spam</h3>
```

Comment:
Not super important, but for consistency with below:

```suggestion
          <h3>{% trans "Spam" %}</h3>
```